modules/calc/src/main/java/com/opengamma/strata/calc/runner/MissingConfigCalculationFunction.java

```diff
@@ -23,13 +24,18 @@
  */
 public class MissingConfigCalculationFunction
     implements CalculationFunction<CalculationTarget> {
-  // this must be public so that the constructor can be invoked
+  // TODO package scoped
 
   /**
    * Shared instance.
    */
   public static final CalculationFunction<CalculationTarget> INSTANCE = new MissingConfigCalculationFunction();
 
+  // restricted constructor
+  public MissingConfigCalculationFunction() {
```

Comment:
This is caused by splitting this commit from the next one. When the next commit is merged, this issue will be solved.